modular_skyrat/modules/positronic_alert_console/code/positronic_alert_console.dm

```diff
@@ -5,13 +5,28 @@
 	icon_state = "posialert"
 	var/inuse = FALSE
 
+/var/radio_key = /obj/item/encryptionkey/headset_sci
+/var/obj/item/radio/radio
```

Comment:
You need to have this indented properly, it works right now because you created two global variables, which obviously is a little stinky.
```suggestion
	var/inuse = FALSE

	/// The encryption key typepath that will be used by the console.
	/var/radio_key = /obj/item/encryptionkey/headset_sci
	/// The radio used to send messages over the science channel.
	/var/obj/item/radio/radio
```

Reply:
Ah I see